src/components/pluginManager.js

```diff
@@ -74,14 +74,23 @@ import { playbackManager } from './playback/playbackmanager';
             if (typeof pluginSpec === 'string') {
                 if (pluginSpec in window) {
                     console.log(`Loading plugin (via window): ${pluginSpec}`);
-                    let pluginInstance = await window[pluginSpec];
 
-                    if (typeof pluginInstance === 'function') {
-                        pluginInstance = await new pluginInstance();
+                    const pluginDefinition = await window[pluginSpec];
```

Comment:
Not sure how that works on the Android side, maybe you already do this, but wouldn't it be a good idea to make the plugins a property of a "plugins" property on Window? (Maybe even `window.jellyfin.plugins` or something)

Just to make window a bit less dirty.

Reply:
I'd be in favor, but I don't know how other clients use the plugins in window.

Reply:
This would probably be doable (I think Android and old desktop are the only clients using these), but it shouldn't be done in this PR since it will be backported as a bugfix.

Reply:
Yes, that should be improved in a future version.